linera-core/src/chain_worker/state/mod.rs

```diff
@@ -306,39 +306,95 @@ where
     }
 
     /// Returns an error if the block requires a blob we don't have.
-    /// Looks for the blob in: chain manager's pending blobs and storage.
-    async fn check_no_missing_blobs(
+    /// Looks for the blob in: chain manager's proposed blobs and storage.
+    async fn get_proposed_blobs(
```

Comment:
Method is called `get_proposed_blobs` but it does something different - please rename to sth that matches the logic.